coriolis_openstack_utils/cli/tenant.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from oslo_log import log as logging
+
+from cliff import lister
+
+from coriolis_openstack_utils import conf
+from coriolis_openstack_utils.actions import tenant_actions
+from coriolis_openstack_utils.cli import formatter
+
+
+LOG = logging.getLogger(__name__)
+
+
+class TenantMigrationFormatter(formatter.EntityFormatter):
+    columns = (
+        "Tenant Name",
+        "Tenant ID")
+
+    def _get_formatted_data(self, obj):
+        data = (
+            obj["name"],
+            obj["id"])
+
+        return data
+
+class MigrateTenant(lister.Lister):
+    def get_parser(self, prog_name):
```

Comment:
We'll need to define args to cover the three base usecases a user might want from the utils (relating to the VM migrations he/she'd like to trigger), and namely:
  * replicate the tenant and all non-VM bits and bobs (networks, secgroups, routers, blabla), but *no* VM
  * replicate the tenant, everything non-VM, and a given set of VM names/IDs.
  * replicate the tenant, including all VMs

Imo the best way to do this is by having 3 mutually exclusive flags like:
```bash
coriolis-openstack-util migrate tenant [...] --all-instances  # let's call this the default so we can skip defining an explicit arg
coriolis-openstack-util migrate tenant [...] --no-instances
coriolis-openstack-util migrate tenant [...] --instances $VM1 $VM2 [...]
```
There may be a smarter way to implement this than a mutually exclusive group, but the point is we need to offer all three of these functionalities to cover all usecases.